### PR TITLE
Button to hide events

### PR DESCRIPTION
**Pull request by OliverBalfour**

### CODE REVIEW

**Comment by OliverBalfour** [comment]
Hmm GetICS is giving internal errors. Is there a way to debug a function @pl4nty?

**Comment by OliverBalfour** [comment]
Ohh I added a `?.` and it's node 12, of course. That'd do it 😛

**Comment by pl4nty** [comment]
> Hmm GetICS is giving internal errors. Is there a way to debug a function @pl4nty?

If you're using the vscode debug tasks, the usual tools should be available - breakpoints, step through, etc

**Comment by pl4nty** [comment]
It also shows the individual Function debug task (part of the Full Stack task), if you just want to debug Functions without the frontend

**Comment by OliverBalfour** [comment]
@pl4nty GetICS now works with hidden events, ready for review and merge

File: src/App.js [hunk 1]
```diff
@@ -55,6 +55,21 @@ let App = () => {
   const [specifiedOccurrences, setSpecifiedOccurrences] = useState(getSpecOccurrences())
   const updateSpecifiedOccurrences = () => setSpecifiedOccurrences(getSpecOccurrences())
 
+  // Events that are manually hidden with the eye icon
+  const [hiddenOccurrences, setHiddenOccurrences] = useState(h)
+  useEffect(() => console.log(hiddenOccurrences))
```
Comment (pl4nty) [inline src/App.js:60]:
```suggestion
```
image

**Review by pl4nty — APPROVED** [review]
Could the button be a touch bigger? LGTM otherwise, nice one!

**Pull request merged**